packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md

```diff
@@ -1 +1,7 @@
-This functionality is coming soon. Watch this GitHub issue for updates: [#126 Support OIDC logout](https://github.com/okta/okta-spring-boot/issues/126)
+You can configure an RP-Initiated (SSO) Logout by setting the `okta.oauth2.postLogoutRedirectUri` property with an absolute URI such as:
```

Comment:
I think our other docs mention that this should be placed in `application.properties`, double-check and make sure the wording is the same

Reply:
You're right! Updated!